.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,7 @@
+{
+  "dockerComposeFile": "../dev/docker-compose.yaml",
+  "service": "devcontainer",
+  "remoteUser": "helpwave",
+  "workspaceFolder": "/helpwave",
+  "overrideCommand": false
+}
```

Comment:
you could utilize the `postCreateCommand` to install dependencies like pre commit hooks or the `initializeCommand` to regularly pull docker images so that they don’t get outdated. 

It could also be helpful to use the `customizations` setting to add extensions or settings for vscode when used with a devcontainer.

Reply:
Great points, and I already did think about them!


> postCreateCommand

We have the entrypoint.sh for that. It runs when using docker (standalone), docker compose and using devcontainers. I want these environments to be as close to each other as possible.

> regularly pull docker images

By pinning the version of the image in git this is done automatically when starting the environment. This way it's also clear which versions of the image work with what states of the codebase. We can checkout old versions and still get a runnning system